### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by arbulu89**
Github issues template. Now that we are receiving more issues, we need to ask for specific information to make the analysis smoother

### CODE REVIEW

**Review by stefanotorresi — CHANGES_REQUESTED** [review]
Instead of "Describe the bug", I would suggest asking for a step-by-step description on how to reproduce it consistently, and also the expected result vs the observed behaviour.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+**Used cloud platform**
+Specify the used cloud platform (AWS, GCP, Azure, etc)
+
+**Used SLES4SAP version**
+Specify the used SLES4SAP version (SLES12SP4, SLES15SP2, etc)
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (MalloZup) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: we can add here on the line

`A clear and concise description of what the bug is and how to reproduce it`

**Review by MalloZup — APPROVED** [review]
looks good to me. Personally I like the sentence how to reproduce it, since it tells the user to specify a step by step, whereas maybe describe the bug could be more "theoretical".

But as far we state it even later on I"m ok.

**Comment by stefanotorresi** [comment]
> 1. You modify the terraform.tvars
> 2. Run terraform commands

That's a big assumption! Users will do all sorts of strange things with software. 😀

**Comment by arbulu89** [comment]
@stefanotorresi @MalloZup Template with new questions updates

**Review by stefanotorresi — APPROVED** [review]
perfect, thank you!

**Pull request merged**